{"name": "test_delete_evidence", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "self = <test_case_details.TestCaseDetails object at 0x0DAED6B0>\n\n    def test_delete_evidence(self):\n        driver = self.driver\n        self.login()\n        case_details = CaseDetails(driver)\n        home = HomePage(driver)\n        home.click_law_suit_page()\n        case_details.evidence_tab_entry()\n>       case_details.delete_evidence_attachment()\n\ntest_case_details.py:59: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <Pages.CaseDetailsPage.CaseDetails object at 0x0DBC4AF0>\n\n    def delete_evidence_attachment(self):\n        self.driver.find_element_by_xpath(self.delete_evidence_attachment_btn_xpath).click()\n        time.sleep(2)\n        # There should be a warning popup\n>       assert \"There should be a warning popup\" == \"There should be a warning\"\nE       AssertionError\n\n..\\Pages\\CaseDetailsPage.py:123: AssertionError"}, "attachments": [{"name": "stdout", "source": "e6a008f8-5b30-489d-b4e9-dfde45c28cf3-attachment.txt", "type": "text/plain"}], "start": 1568271026278, "stop": 1568271047895, "uuid": "5ab47789-4368-4126-9ccc-bcdb4a57906f", "historyId": "fda4845cb7aa8152e3b454e384844763", "testCaseId": "c8a0df3bb95de312caaaeaafdd5f535f", "fullName": "test_case_details.TestCaseDetails#test_delete_evidence", "labels": [{"name": "parentSuite"}, {"name": "suite", "value": "test_case_details"}, {"name": "subSuite", "value": "TestCaseDetails"}, {"name": "host", "value": "EVT01066NB"}, {"name": "thread", "value": "19116-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case_details"}]}